### PR TITLE
Fix incorrect package name in `pkg/api/cluster/formatter.go`

### PR DESCRIPTION
**Pull request by brandboat**
**Problem:**
execute `make` and I got error
```text
pkg/api/cluster/formatter.go:14:2: cannot find module providing package github.com/rancher/wrangler/pkg/generated/controllers/core/v1: import lookup disabled by -mod=vendor
	(Go version in go.mod is at least 1.14 and vendor directory exists.)
FATA[0011] exit status 1
make: *** [Makefile:11: default] Error 1
```

**Solution:**
Fix incorrect package in `pkg/api/cluster/formatter.go`, the root cause is we change the rancher wrangler version in https://github.com/harvester/harvester/pull/6173

**Related Issue:**
N/A

**Test plan:**
Compile should pass.

### CODE REVIEW

**Review by Yu-Jack — APPROVED** [review]
LGTM, thanks! Wait for CI passed.

**Review by FrankYang0529 — APPROVED** [review]
LGTM. Thanks for the quick fix.

**Comment by FrankYang0529** [comment]
@ibrokethecloud, could you cherry-pick the fix to https://github.com/harvester/harvester/pull/6238? Or do we prefer to merge https://github.com/harvester/harvester/pull/6238 first and then backport this again to v1.4? Thanks.

**Comment by ibrokethecloud** [comment]
@FrankYang0529 i am ok with either option. I can rebase the backport PR once this is merged.

**Pull request merged**
